src/plugins/index_pattern_management/public/components/edit_index_pattern/source_filters_table/components/header/header.tsx

```diff
@@ -28,18 +28,17 @@ export const Header = () => (
       <h3>
         <FormattedMessage
           id="indexPatternManagement.editIndexPattern.sourceHeader"
-          defaultMessage="Source filters"
+          defaultMessage="Field filters"
         />
       </h3>
     </EuiTitle>
     <EuiText>
       <p>
         <FormattedMessage
           id="indexPatternManagement.editIndexPattern.sourceLabel"
-          defaultMessage="Source filters can be used to exclude one or more fields when fetching the document source. This happens when
-          viewing a document in the Discover app, or with a table displaying results from a saved search in the Dashboard app. Each row is
-          built using the source of a single document, and if you have documents with large or unimportant fields you may benefit from
-          filtering those out at this lower level."
+          defaultMessage="Field filters can be used to exclude one or more fields when fetching a document. This happens when
+          viewing a document in the Discover app, or with a table displaying results from a saved search in the Dashboard app.
+          If you have documents with large or unimportant fields you may benefit from filtering those out at this lower level."
```

Comment:
@mattkime @timroes @ppisljar 

We need to discuss if we are comfortable with this change. Also there is a note a few lines below this (L47-48) that reads:

> "Note that multi-fields will incorrectly appear as matches in the table below. These filters only actually apply to fields in the original source document, so matching multi-fields are not actually being filtered."

However, multi-fields _will_ be filtered now for fields provided to `.setField('fields')` as this filtering happens on the client. So it's just `.setField('fieldsFromSource')` where they are not filtered.

Any ideas on how we should address this?

Reply:
^^ Bumping this -- does anybody have any concerns around changing this wording? I don't want this change to slip in unnoticed in case folks have ideas of better ways to handle this.

cc @elastic-jb As this represents a slight change to how the index patterns "source filters" feature is presented to end users